client/src/components/App/ColorPicker.js

```diff
@@ -43,9 +44,9 @@ class ColorPicker extends PureComponent {
 
     render() {
         return (
-            <span style={{ color: this.state.color }}>
+            <Fragment>
```

Comment:
Nit: use `<>` instead of fragment
This is just a shorthand for React.Fragment (literally does the same thing), but it is more concise and doesn't require an import.
```suggestion
            <>
```

Reply:
A bunch of other places in our code use Fragment, so I thought we were using it just to be more explicit. Should we open an issue to refactor all instances of Fragment to <>?

Reply:
Yes please!

Reply:
ok im on it